test/mocks/data/firestore_user_comment.dart

```diff
@@ -0,0 +1,53 @@
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/comment/comment_id_firestore.dart";
+import "package:proxima/models/database/user/user_id_firestore.dart";
+import "package:proxima/models/database/user_comment/user_comment_data.dart";
+import "package:proxima/models/database/user_comment/user_comment_firestore.dart";
+import "package:proxima/services/database/user_comment_repository_service.dart";
+
+import "user_comment_data.dart";
+
+/// This class is responsible for generating mock [UserCommentFirestore] instances.
+class UserCommentFirestoreGenerator {
+  int _commentId = 0;
+  final UserCommentDataGenerator _userCommentDataGenerator;
+
+  UserCommentFirestoreGenerator({int seed = 0})
+      : _userCommentDataGenerator = UserCommentDataGenerator(seed: seed);
+
+  /// Add [number] of user comments for the user with id [userId] to the firestore database.
+  /// It will use the [userCommentRepository] to add the comments.
+  /// It will return the list of user comments that were added.
+  Future<List<UserCommentFirestore>> addComments(
+    int number,
+    UserIdFirestore userId,
+    UserCommentRepositoryService userCommentRepository,
+  ) async {
+    final userComments = <UserCommentFirestore>[];
+
+    for (var i = 0; i < number; i++) {
+      final userComment = createMockUserComment();
+      userComments.add(
+        UserCommentFirestore(id: userComment.id, data: userComment.data),
+      );
+
+      await userCommentRepository.addUserComment(userId, userComments.last);
+    }
+
+    return userComments;
```

Comment:
I feel like this implementation is quite inefficient: repetedly adding elements to a list is expensive, and waiting for the current comment to be added before moving onto the next one is not really efficient. I don't know if this is so important since this is a test, and maybe it is not a big bottleneck for test speed. I however suggest a differnet implementation, such as:
```suggestion
    final userComments = List.generate(number, (_) => createMockUserComment());
    await Future.wait(
      userComments.map(
        (comment) =>
            userCommentRepository.addUserComment(userId, userComments.last),
      ),
    );
    return userComments;
```

Note that I haven't tested this code, so I may have made a mistake.

Reply:
As you said, I don't think that this is a very big deal given that this is for testing purposes.
But you implementation seems to be more concise thus I'll implement it :+1: 

Done in 143235bd5b1c3152d5158c343a883aa0e65216f7.